app/lib/common/widgets/chat/convo_card.dart

```diff
@@ -261,28 +253,21 @@ class _SubtitleWidget extends ConsumerWidget {
           mainAxisSize: MainAxisSize.min,
           children: [
             Flexible(
-              child: Padding(
-                padding: const EdgeInsets.symmetric(vertical: 10),
-                child: Text(
-                  '${simplifyUserId(sender)}: ',
-                  style: Theme.of(context)
-                      .textTheme
-                      .bodySmall!
-                      .copyWith(fontWeight: FontWeight.w700),
-                  overflow: TextOverflow.ellipsis,
-                ),
+              child: Text(
+                '${simplifyUserId(sender)}: ',
+                style: Theme.of(context)
+                    .textTheme
+                    .labelMedium!
+                    .copyWith(fontWeight: FontWeight.w700),
+                overflow: TextOverflow.ellipsis,
               ),
             ),
             Flexible(
               child: Html(
-                padding: const EdgeInsets.symmetric(vertical: 10),
                 // ignore: unnecessary_string_interpolations
                 data: '''$body''',
                 maxLines: 1,
-                defaultTextStyle: const TextStyle(
-                  overflow: TextOverflow.ellipsis,
```

Comment:
Same goes here. The overflow manages the text part based on screen width and its removed...

Reply:
Resolved by https://github.com/acterglobal/a3/pull/1974/commits/6b0537a78daba6b6bac6f53a5538ee6f71bbb4dd